app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,11 @@
+<% provide(:title, "About") %>
+<h1>About</h1>
+<p>
+  The <a href="http://www.railstutorial.org/"><em>Ruby on Rails
```

Comment:
đổi thẻ a thành link_to, em tìm và sửa ở toàn bộ pull nhé, cũng giống với i18n, chỗ nào có thể dịch được thì áp dụng luôn